sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/BigDataPoolsClientLiveTests.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Azure.Analytics.Synapse.Artifacts;
+using Azure.Analytics.Synapse.Artifacts.Models;
+using Azure.Analytics.Synapse.Tests;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Analytics.Synapse.Artifacts.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="BigDataPoolsClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class BigDataPoolsClientLiveTests : RecordedTestBase<SynapseTestEnvironment>
+    {
+        public BigDataPoolsClientLiveTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        private BigDataPoolsClient CreateClient()
+        {
+            return InstrumentClient(new BigDataPoolsClient(
+                new Uri(TestEnvironment.EndpointUrl),
+                TestEnvironment.Credential,
+                InstrumentClientOptions(new ArtifactsClientOptions())
+            ));
+        }
+
+        [Ignore("This test case cannot be automated due to the inability to configure infrastructure to test against.")]
```

Comment:
There's a test category "Manually" that seems to be the recommended approach for these types.   Tests with that category are not run automatically by the pipelines.

Reply:
I believe most of these could be automated, I don't want to give up yet. Filing an issue and using it to track.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/18080